packages/devtools/web/styles.css

```diff
@@ -39,7 +39,7 @@
     --subtle: #777;
     --table-border: #dfe2e5;
     --title: #555;
-    --toast-background: f9f9f9;
+    --toast-background: #f9f9f9;
```

Comment:
This should be a css variable to work with the dark theme.

Reply:
This is a variable, but it was missing the `#` in the light theme so it didn't work (the toast was transparent and you could see the contents underneath through it) :-)